tests/roles/development_environment/defaults/main.yaml

```diff
@@ -1,5 +1,17 @@
 prelaunch_test_instance: true
 prelaunch_test_instance_script: pre_launch.bash
+# Ping test activation flag, note that it needs
+# prelaunch_test_instance as well.
+ping_test: true
```

Comment:
I was wondering whether we want to default it to true or false, i guess either option is fine, just has different benefits and drawbacks.

Reply:
With the current thresholds, it makes sense to keep them true by default, so we can collect logs.
In 18 update, we have it to false and only activated for the update job, so it doesn't affect other group jobs and they can activate it as they see fit.  Not sure if we affect other groups here.

Reply:
Yea thats' a fair point, we do affect others & dev envs. So it probably does make sense to switch it to false here and propose companion patches to CI to enable it there.

Reply:
So with the current threshold nobody should have any error but everybody will have logs and if something insane like > 900 m cut and 50% loss, there a error will appear.

So another approach would be to lower the threshold in our job when the fix is there.  And then propose to other group to have lower threshold as well.

It seems to provide best of both world : everybody gets logs, nobody get failure until they opt-in for stringent threshold.

And if cut of 900m and 50% then ... well failure seems like a good idea.  Or we can maxout everything with 9000s and 100%, and re-inforce the "opt-in" idea.

But yeah, another approach to activate things that is less on/off type of things.

Reply:
Oki, that's convincing.  I'll set it to false then.  Should I keep those insane threshold or should we set them in the job as well, until the problem is resolved.  I think we should put the intend here (so 0 ping loss) and override it in the job. WDYT?

Reply:
+1 on your suggestion. If we want the defaults here edited, we can do so easily later. But each job should probably be setting its expectation.